src/java.desktop/share/classes/javax/swing/ImageIcon.java

```diff
@@ -61,6 +61,11 @@
  * from Images. Images that are created from a URL, filename or byte array
  * are preloaded using MediaTracker to monitor the loaded state
  * of the image.
+ * If the image source parameter to a constructor is non-null,
```

Comment:
Whilst reviewing the CSR I realized you need to change the line above to say
"If the image source parameter to a constructor or method is non-null,"

I made that change in the CSR. Do it here too. It is needed to cover setImage()

Reply:
Added